src/main/java/com/jagrosh/jmusicbot/commands/dj/MoveTrackCmd.java

```diff
@@ -0,0 +1,92 @@
+package com.jagrosh.jmusicbot.commands.dj;
+
+
+import com.jagrosh.jdautilities.command.CommandEvent;
+import com.jagrosh.jmusicbot.Bot;
+import com.jagrosh.jmusicbot.audio.AudioHandler;
+import com.jagrosh.jmusicbot.audio.QueuedTrack;
+import com.jagrosh.jmusicbot.commands.DJCommand;
+import com.jagrosh.jmusicbot.queue.FairQueue;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+/**
+ * Command that provides users the ability to move a track in the playlist.
+ */
+public class MoveTrackCmd extends DJCommand
+{
+
+    private static final Pattern PATTERN = Pattern.compile("^(\\d+)\\s+(\\d+)$");
```

Comment:
I'm not a huge fan of using a regex here; I'd rather split on whitespace first and then attempt to parse integers (take a look at the other commands that accept numerical inputs)